en/config_rover/mecanum.md

```diff
@@ -69,7 +68,6 @@ To set up [Acro mode](../flight_modes_rover/mecanum.md#acro-mode) navigate to [P
    5. Move the right-stick of your controller all the way to the left or right and observe the behaviour of the rover.
       Keep reducing the value of [RM_MAX_YAW_RATE](#RM_MAX_YAW_RATE) until you are satisfied with the maximum turn rate.
 
-   Remember to reset [RM_MAN_YAW_SCALE](#RM_MAN_YAW_SCALE) back to its original value.
```

Comment:
@chfriedrich98 For `RM_MAN_YAW_SCALE` I have just assumed that you don't need to reset the new values to anything - ie. RM_MAX_YAW_RATE

Can you sanity check the changes where RM_MAX_YAW_RATE has been removed?

Reply:
LGTM